web/checks_api_test.go

```diff
@@ -120,11 +120,11 @@ func TestApiClusterCheckResultsHandler(t *testing.T) {
 	assert.Equal(t, 200, resp.Code)
 }
 
-func TestApiClusterCheckResultsHandler500(t *testing.T) {
+func TestApiGetCheckResultsHandler404(t *testing.T) {
 	mockChecksService := new(services.MockChecksService)
 	mockChecksService.On(
-		"GetChecksResultAndMetadataByCluster", "47d1190ffb4f781974c8356d7f863b03").Return(
-		&models.ClusterCheckResults{}, fmt.Errorf("kaboom"))
+		"GetChecksResultAndMetadataById", "47d1190ffb4f781974c8356d7f863b03").Return(
```

Comment:
I've noticed that sometimes you use "CheckResults" and others you move the plural: "ChecksResult". Is this intentional?

Reply:
Also sometimes I see "ChecksResults". There might be a reason I'm missing but if not, I'd try to keep it constant across the code.